services/backend/src/database/migrations/migrate.lock

```diff
@@ -1,4 +1,5 @@
 # Prisma Migrate lockfile v1
 
 20201019154547-init
-20201109171010-create-user-bugs
\ No newline at end of file
+20201027192648-add-qualified-pools-table
+20201109171010-create-user-bugs
```

Comment:
This may cause a problem, you may need to recreate a new migration (since the dev db will have the latest migration, but the migration before) - I will need to test it locally and will report back in this comment